.travis.yml

```diff
@@ -0,0 +1,15 @@
+language: python
+
+python:
+  - "2.7"
+
+before_install:
+    - pip install pep8
+
+install:
+    - python setup.py -q install
+
+script:
+    # Don't fail if a line is longer than 80 characters (E501)
+    - find . -name \*.py -exec pep8 --ignore=E501 --exclude='*/procedural_city_generation/*' {} +
```

Comment:
I agree in 79-char lines for most of the cases, but there are some places where it is a pain (e.g. having to split a url https://github.com/ekumenlabs/terminus/blob/master/terminus/generators/rndf_generator.py#L24-L25). I fear however that if we don't enforce it we will eventually get sloppy but lets give it a try. I'll keep running full pep8 for a week or so and see if the exceptions to E501 are ok. If not, we can always enforce it back and fix the offending lines, it will take just a few moments.